debezium-core/src/main/java/io/debezium/transforms/debeziumtimestampcoverter/DebeziumTimestampConverter.java

```diff
@@ -0,0 +1,698 @@
+    private static final String FIELD_DEFAULT = "";
+
+    public static final String TARGET_TYPE_CONFIG = "target.type";
+
+    public static final String TIMESTAMP_FORMAT_CONFIG = "timestamp.format";
+    private static final String TIMESTAMP_FORMAT_DEFAULT = "yyyy-MM-dd HH:mm:ss";
+
+    public static final String DATE_FORMAT_CONFIG = "date.format";
+    private static final String DATE_FORMAT_DEFAULT = "yyyy-MM-dd";
+
+    public static final String FIELD_TYPE_CONFIG = "field.type";
+    private static final String FIELD_TYPE_DEFAULT = "empty.value";
+
+    public static final String STRUCT_FIELD_NAME_CONFIG = "struct.field";
+    private static final String STRUCT_FIELD_NAME_DEFAULT = "";
+
+
+    public static final ConfigDef CONFIG_DEF = new ConfigDef()
+            .define(FIELD_CONFIG, ConfigDef.Type.STRING, FIELD_DEFAULT, ConfigDef.Importance.LOW,
+                    "The field containing the timestamp, or empty if the entire value is a timestamp")
+            .define(TARGET_TYPE_CONFIG, ConfigDef.Type.STRING, ConfigDef.Importance.LOW,
+                    "The desired timestamp representation: string, unix, Date, Time, or Timestamp")
+            .define(TIMESTAMP_FORMAT_CONFIG, ConfigDef.Type.STRING, TIMESTAMP_FORMAT_DEFAULT, ConfigDef.Importance.LOW,
+                    "A SimpleDateFormat-compatible format for the timestamp. Used to generate the output when type=string "
+                            + "or used to parse the input if the input is a string.")
+            .define(DATE_FORMAT_CONFIG, ConfigDef.Type.STRING, DATE_FORMAT_DEFAULT, ConfigDef.Importance.LOW,
+                    "A SimpleDateFormat-compatible format for the date. Used to generate the output when type=string "
+                            + "or used to parse the input if the input is a string.")
+            .define(FIELD_TYPE_CONFIG, ConfigDef.Type.STRING, FIELD_TYPE_DEFAULT, ConfigDef.Importance.LOW, "is a comma seperated string, providing an option to add multiple converters.\n" +
+                    "<Type of Timestamp Object> -> <To Target type>\n" +
+                    "can add multiple type converters as below," +
+                    " <Type of Timestamp Object> -> <To Target type>,<Type of Timestamp Object> -> <To Target type>")
+            .define(STRUCT_FIELD_NAME_CONFIG, ConfigDef.Type.STRING, STRUCT_FIELD_NAME_DEFAULT, ConfigDef.Importance.LOW, "Field name that is only to be modified in Nested Struct");
+
+    private static final String PURPOSE = "converting timestamp formats";
+    private static final String TYPE_INT64 = "INT64";
+    private static final String TYPE_INT32 = "INT32";
+    private static final String TYPE_INT16 = "INT16";
+    private static final String TYPE_INT8 = "INT8";
+    private static final String TYPE_FLOAT32 = "FLOAT32";
+    private static final String TYPE_FLOAT64 = "FLOAT64";
+    private static final String TYPE_BOOLEAN = "BOOLEAN";
+    private static final String STRING_TYPE = "STRING";
```

Comment:
i am using an other STRING_TYPE variable, in the code, because of which i had to name this one as STRING_TYPE.
But i think i can maintain a single STRING_TYPE(lower case string) and can convert to upper case wherever required, WDYT?

Reply:
IMO, I think the above variables should be consistent. If you have a lowercase version, then that variable should be `TYPE_STRING_LOWERCASE` or something like that.